Person/persons.md

```diff
@@ -0,0 +1,386 @@
+*lattitude* | `String` | lattitude of the location of the person
+*longitude* | `String` | longitude of the location of the person
+
+
+```console
+POST person/:target_person_uuid/address
+{ 
+    "address1": "30, Vivekananda Layout, Munnekolal,Marathahalli",
+    "cityVillage": "Bengaluru"
+    "stateProvince": "Karnataka",
+    "postalCode": "560037",
+    "lattitude": "28.65033",
+    "longitude": "77.304255"
+}
+```
+
+### Delete a person address sub resource
+
+* Delete or retire a target address sub resource by its UUID.Returns a `404 Not Found` status if attribute not exists. 
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+ ```console
+DELETE /person/:target_person_uuid/person/:target_address_uuid
+ ```
+
+### List person attribute subresource
+
+* List all the person attribute subresource corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person attribute not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+
+```console
+GET /person/:target_person_uuid/attribute
+```
+
+* List all the person attribute subresource by it's `UUID` and corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person attribute not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+
+```console
+GET /person/:target_person_uuid/name/:target_attribute_uuid
+```
+
+### Create person attribute subresource
+
+* To create a person attribute sub resource for a specific person resource you need to specify below properties in your request body.
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+#### Attributes
+
+Parameter | Type | Description
+--- | --- | ---
+*attributeType* | `UUID` | UUID of the attributeType
+*value* | `String` | value associated with the attribute
+*hydratedObject* | `UUID` | UUID of the hydrated person attribute object
+
+```console
+POST person/:target_person_uuid/attribute
+{ 
+    "attributeType": "8d8718c2-c2cc-11de-8d13-0010c6dffd0f",
+    "value": "Birthplace",
+}
+```
+
+### Update person attribute subresource
+
+* To update a person attribute with given uuid value for a specific person resource you need to specify below properties in your request body.
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+#### Attributes
+
+Parameter | Type | Description
+--- | --- | ---
+*attributeType* | `UUID` | UUID of the attributeType
+*value* | `String` | value associated with the attribute
+*hydratedObject* | `UUID` | UUID of the hydrated person attribute object
+
+```console
+POST person/:target_person_uuid/attribute
+{ 
+    "attributeType": "8d8718c2-c2cc-11de-8d13-0010c6dffd0f",
+    "value": "Birthplace",
+}
+```
+
+### Delete a person attribute sub resource
+
+* Delete or retire a target attribute sub resource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
```

Comment:
Person attributes are [voided](https://docs.openmrs.org/datamodel/openmrs-refapp-data-model-1.9.html#person_attribute.void), not retired.